.github/workflows/ci.yaml

```
@@ -120,14 +123,20 @@ jobs:
       - name: Install dependencies
         run: composer install --no-interaction --no-suggest --no-progress --no-scripts
 
+      - name: Run Migrations
+        run: php artisan migrate --force --seed
+        env:
+          DB_PORT: ${{ job.services.database.ports[3306] }}
```

actionlint (1.7.10)

property access of object must be type of string but got "number"

(expression)